NAMESPACE

```diff
@@ -3,3 +3,7 @@ import(RJSONIO)
 importFrom(utils, packageDescription)
 
 exportPattern("^[[:alpha:]]+")
+
+S3method("pbGetDevices", "default")
```

Comment:
S3method() != export().  It's to declare that a function with name `<name>.<class>` is an S3 method `<name>` for class `<class>`.  This helps S3 and S4 method dispatching to work properly.  This is the only way R can know that, e.g., `t.test()` is not method `t` for class `test`.